README.md

```diff
@@ -1,53 +1,73 @@
 # cloud-platform-environments
 
 ## Intro
-
-This repo contains the necesary files to create a pipeline in aws to create and delete kubernetes cluster namespaces and resources after a push is made to the master branch.
-The aws resources and pipeline can be created with the terraform templates included. One Pipeline exist for each cluster under the `terraform` directory.
-Kubernetes namespaces and resources are defined in the namespaces directory in this repo under the corresponding cluster name.
+This repository contains the necessary files to create a pipeline in AWS to create and delete kubernetes cluster namespaces and resources after a push is made to the master branch. The AWS resources and pipeline can be created with the terraform templates included. One Pipeline exist for each cluster under the `terraform` directory. Kubernetes namespaces and resources are defined in the `namespaces` directory in this repository under the corresponding `cluster` name.
 
 ### Functionality
+The pipeline will for each defined `cluster`:
 
-The pipeline will for each defined cluster:
-1) create any namespace defined in the `namespaces/<cluster>` directory in the corresponding cluster that does not actually exist on the cluster.
-2) delete any namespaces that exist in the <cluster> but is not defined in the repo.
-3) create any kubernetes resource that is defined under `namespaces/<cluster>`
+1) create any namespace defined in the namespaces/`cluster` directory in the corresponding cluster, that does not actually exist on the cluster.
```

Comment:
This is difficult to understand. Perhaps: 
"Create a namespace defined the namespace/`cluster` directory. If the namespace already exists on the cluster it will be ignored."